config/config_test.go

```diff
@@ -444,6 +446,7 @@ leader_election:
 					MaxPointAge: DurationConfig{
 						72 * time.Hour,
 					},
+					HealthCheckRequestTimeout: DurationConfig{config.DefaultHealthCheckTimeout},
```

Comment:
maybe for this test we add an example of parsing the health check request timeout instead of default so we're covering both default and custom case